doc/ipython-notebooks/data_project/health_data_project.ipynb

```diff
@@ -0,0 +1,2248 @@
tavqsNqslgRqmJQ30h+MjyTDHW0FhERERG5aArKIh2MMQbz\njyfh2GFs9/w/rJ5ZAHj8Qbbn17P7uIeoEjupOClO9fGjCZn0jFNAFhERERG5VBSURToYs+ptzMaV\nWAtuombIBDYfrObDE40EaqAnLvpakRjLkD3SxXVDEsNdroiIiIhIl6OgLNKBhDauovbV51k99RsU\nM4KIN6tIwUG2FUnIbohLsdOvj5u0DCcRkdoTWURERESkLSgoi3QQZucmTrz+Dutm/JpkWxS9vAYT\nBWk9nQzIdpOY5MCyaS9kEREREZG2pqAs0gGYA7vYvngHx8f/mHjLRspAO+OHReN0adZYRERERKS9\nKSiLhFnww4OsWlNK04Av4bcFGXtZJDk9I8JdloiIiIhIt6WgLBJGng+PsmKjDVvqJEqcTdw4L5nE\nGGe4yxIRERER6dYUlEXCpGh/IVv2RhOKSKAopp47ruyJy66l1iIiIiIi4XZRQTkYDGK32y9VLSLd\nQihkOLClgmMnIqmxBbCl1XLnrD7YLDXqEhERERHpCM47KJeWlrJ9+3YOHjzI8ePHqampwefz4Xa7\niYuLIzs7m2HDhjFu3Dh69OjRljWLdFp+v+H9NVXUVDk4bBoY2reRqyb3D3dZIiIiIiLyCZ8alHfu\n3MmyZcvYu3cvxpgzjnu9XsrKyigrK2Pbtm08++yzjBo1iiuvvJIxY8a0SdEinVFTY4j3V9dQW2ex\nJVDODYMDTBg/ONxliYiIiIjIv2k1KB8/fpxnn32WgwcPAjBgwACGDRvGgAEDyMzMJCYmhqioKDwe\nD/X19eTn53P48GH279/P7t272b17N0OHDuW2224jOzu7ve5HpEOqqwmyZW0tHk+Ad00dt/b3MmH8\nsHCXJSIiIiIiZ9FqUH7wwQeJjIzkhhtuYNasWaSnp5/1vJiYGGJiYkhPT2f8+PEAFBUVsWbNGlas\nWMGDDz7ICy+80DbVi3QCFWUBtq2vI9TUwFumkeGpQSZPVkgWEREREemoWg3K1113Hddccw0xMTEX\n/KEZGRnccsstXHvttbz99tsXVaBIZ1Z4yseu9xuI8JSxJFiFiY/na7MHhLssERERERE5h1b3orn5\n5ps/U0j+pJiYGG6++eaL+gyRzuroYS87NjUQX3uU49U7yIuI5xvTehPtUqd4EREREZGOTJu2ilxi\nxhgO7mnkwK5G0ir2EFO0mLdTx/C5AQmMzogOd3kiIiIiIvIpLnlQLisr46mnnrrUHyvSKRhj2L3F\nw5FcL30K1zCk8BX+OGwhaTFObh+rbdNERERERDqD895H+dOUlpby2muvsX79eoLBIHfdddel+miR\nTuPwgSbyT/gZcPwN+jds46n536PsZBO/vCKTCIcWcIiIiIiIdAafGpQ3bdrErl27qKmpIS4ujnHj\nxjF58mQsywKgqqqKF198sSUgA4wbN65tqxbpgAoO13D4gCGzcD39m3ax57afsnxzFTcMSWJIj6hw\nlyciIiIiIuep1aAcCoX41a9+xc6dO08b37BhA1OnTuVb3/oWmzdv5sknn6SxsRGAESNGsHDhQvr3\n79+2VYt0ICYUomrN++wuHUBi7XGG96qi8au/4A8ri8mKd3HLqJRwlygiIiIiIheg1aC8evXqlpA8\natQoevfuTWNjI/v27WPTpk2kpqby9ttvEwqFyMnJ4Utf+hJDhw5tt8JFOgJz6hieF/7O9h5fwu1o\nZPwVaTj7TeT3GwupaQrw41nZuOxaci0iIiIi0pm0GpTfe+89AO6//34mTZrUMh4IBPj1r3/Nm2++\nCcCCBQv48pe/3LIUW6Q7MI0ezFvPE1j7LjvG/5hAZDzTrognItHBppO1rDtey80jU8hJigh3qSIi\nIiIicoFaDcqnTp2iX79+p4VkAIfDwcKFC9mxYwcpKSkKydLthLa9h3nxaUxtFftmP0yNrQ8TpkUT\nn+ig1hvkya0l5CRFcOOw5HCXKiIiIiIin0GrQbmhoYGMjIyzHuvZsycAOTk5CsnSbZhAAPPS05g1\nS6BPf47e8AsK86MZPCKC9F5OAP5nRwn1viD/eXlvHDb93xARERER6YxaDcrGGByOsx/+aDwiQstK\npXsw9bWEnnwcDu3DmncDxZNv5tCmJnr1cdJ/iBuAnYX1rDlWy03Dk8lO1P8NEREREZHO6pLtoyzS\nVZmCk4T+8AhUV2B95dvUDZnJ7tV1JCTZGTUhCsuyaPSH+NPWYjLjXNw0XEuuRUREREQ6s3MG5T17\n9vDwww9f8HHLsvjpT3968dWJhJnZs5XQX/4bIiKwPfAo/syBbFteh9NpMeGyaOz25uXVz+8to7Qh\nwC+vyMKpLtciIiIiIp3aOYNydXU11dXVn/m4SGdljMEsfQXzxnOQlYPt6z+ExGT2ve+hqdFw2eUx\nREQ2B+LD5Y28c6iKzw1IYGiPqDBXLiIiIiIiF6vVoHzPPfe0Zx0iHYbxeTHP/h6zdT3WxBlYt30D\ny+Um/7iPwlN+Bo+IICG5+b+OP2j4w5ZiEiMc3DomNcyVi4iIiIjIpdBqUJ41a1Y7liHSMZiGotx3\nhwAAIABJREFUekK/+SmcPIL1f27FuvLzWJaFpyHEvp0eklLs9B/sbjn/9Q8qOFHt5YczexHltIex\nchERERERuVTUzEvkE8zKN+FEHrZ7f4g1enLzWMiwe0sDGBgzKQrrX9s+5dd4eXFfBdOyYpmUGRvO\nskVERERE5BL61KC8d+9eNm/eTHl5OQ6Hg+zsbObMmUNKSkp71CfSbozPi1m7FEZNbAnJAEcOe6ko\nCzJ6YiRRMc2zxiFj+OOWYiIcFneOTwtXySIiIiIi0gbOGZT//Oc/s2rVqtPGduzYwdtvv833vvc9\nRo4c2abFibQns3kt1Ndiu+K6lrGaqiC5+5pIz3SSme1qGX/3w2oOljXyjcnpJERqY
```

Comment:
pls comment here that you are using a pre-written sql statement that is long and messy :)